src/handbook/operations/billing.md

```diff
@@ -10,10 +10,14 @@ their role with an appropriate level of access.
 
 We also contract with Formation Financial to help assist with business administration and billing support. You can direct any billing related requests to billing@flowfuse.com for their assistance if preferred.
 
+You should first get approval from the CEO or CTO to make any changes to existing subscriptions.
```

Comment:
```suggestion
You should first get approval from the CEO or CTO to make any changes to existing subscriptions. Use coupons as described below when providing customer access to resources subscribed to through pre-paid deals.
```

Reply:
@zackwasli Just to make sure you know that if you keep the subscription in place we're fine with adding coupons yourself. Let me know if this works for you